src/layers/microscopy-viewer-layer/microscopy-viewer-layer-base.js

```diff
@@ -18,7 +17,7 @@ const defaultProps = {
     } = props.tile;
     const { sliderValues, data, colorValues } = props;
     const xrl = new XRLayer(props, {
-      id: `XR-Layer-${west}-${south}-${east}-${north}`,
+      id: `XR-Layer-${west}-${south}-${east}-${north}-${props.useTiff}`,
```

Comment:
Include `useZarr` as well? I remember you saying we couldn't reduce it to a single boolean, because we may have sources which are neither tiff nor zarr.